pyproject.toml

```diff
@@ -118,7 +118,8 @@ target-version = ["py37", "py38", "py39", "py310"]
 
 [tool.ruff]
 line-length = 120
-select = [
+extend-select = [
```

Comment:
This already contains the default E and F rules, so extend-select should not be used.